### DISCUSSION TITLE
How to Suppress the unwanted errors and How to access the codechecker server while in the same network but different machines?

### DISCUSSION THREAD

**Discussion by shreesh52** [Ericsson/codechecker#4145] [Q&A]
I wanted to suppress error of cppcoreguidelines-avoid-magic numbers for some errors and and some errors I wanted to keep; The other thing is I deployed the server in my localhost and if my friend is also in the same network how can he access my server through ip? It is coming as not found...

**Comment by bruntib** [comment]
Hi,
There shouldn't be a problem with accessing a CodeChecker server. A server should be available through the IP address / hostname and port number where it had been started. This address data can be provided to command line interface too, with `--url` flag,
If the connection still doesn't work, then try starting the server with this flag: `CodeChecker server --not-host-only`.

**Reply by shreesh52** [reply]
Thank you

**Reply by shreesh52** [reply]
It is not working in ubuntu actually

**Reply by bruntib** [reply]
Unfortunately, this is not too much information about the problem, I can't tell why a user can't browse CodeChecker server on the same network if the server was started with `--not-host-noly` flag.
However, I remember a case when a similar situation happened due to proxy settings. Please, check if proxy is set in that environment.
